scripts/zones/Lower_Jeuno/Zone.lua

```diff
@@ -81,16 +84,42 @@ end;
 
 function onGameHour()
     local VanadielHour = VanadielHour();
-
+    local playerOnQuestId = GetServerVariable("[JEUNO]CommService");
+    local playerOnQuest = GetPlayerByID(playerOnQuestId);
+    
     -- Community Service Quest
-    if (VanadielHour == 1) then
-        if (GetServerVariable("[JEUNO]CommService") == 0) then
-            GetNPCByID(17780880):setStatus(0); -- Vhana Ehgaklywha
-            GetNPCByID(17780880):initNpcAi();
-        end;
+    -- 7AM: it's daytime. turn off all the lights
+    if (VanadielHour == 7) then
+        for i=0,11 do
+            local lamp = GetNPCByID(lampIdOffset + i);
+            lamp:setAnimation(ANIMATION_CLOSE_DOOR);
+        end
 
-    elseif (VanadielHour == 5) then
+    -- 8PM: make quest available
+    elseif (VanadielHour == 20) then        
         SetServerVariable("[JEUNO]CommService",0);
+        -- TO DO: notify anyone with Lamp Lighter's Membership Card that it's time
```

Comment:
can modify luautils::onGameHour to push zone object and then loop over all players in zone
```
local players = zone:getPlayers();
for name, player in pairs(players) do
    -- do stuff;
end
```